recipes/read-roi/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set name = "read-roi" %}
+{% set version = "1.0.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/hadim/read_roi
+  git_rev: {{ version }}
```

Comment:
I am adding this to the docs, see https://github.com/conda-forge/conda-forge.github.io/pull/279